clusterfinder/src/main/resources/templates/clusterSearch.ftl

```diff
@@ -655,9 +657,17 @@
 
       showResults: function (data, separator) {
         const clusters = [];
+        const totalFreqency = () => {
+          let tempNo = 0;
+          for (let i = 0; i < data.length; i++) {
```

Comment:
Seda võiks ilusamalt ja lihtsamalt teha, näiteks: https://stackoverflow.com/questions/23247859/better-way-to-sum-a-property-value-in-an-array